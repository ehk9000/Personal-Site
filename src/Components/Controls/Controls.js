import React  from 'react';
import { Route, Switch } from 'react-router-dom';
import Splash  from '../Splash/Splash';
import About from '../About/About';


const Controls = () => {
  return (
    <div>
      <Switch>
        <Route exact path="/" component={Splash}></Route>
        <Route exact path='/About' component={About}></Route>
        <Route exact path='/Projects' component={Splash}></Route>
        <Route exact path='/Experience' component={Splash}></Route>
      </Switch>
    </div>

  );
}

export default Controls;