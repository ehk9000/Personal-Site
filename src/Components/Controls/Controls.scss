.active {
  border-bottom: 2px solid $primary-color;
  padding: 5px;
}