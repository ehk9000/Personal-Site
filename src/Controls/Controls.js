import React  from 'react';
import { Route, Switch } from 'react-router-dom';
import { Splash } from '../Splash/Splash';


const Controls = () => {
  <div>
    <Switch>
      <Route exact path="/" component={Splash}></Route>
    </Switch>
    
  </div>
}

export default Controls;