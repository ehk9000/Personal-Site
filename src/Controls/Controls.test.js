import React from 'react';
import { shallow } from 'enzyme';
import Controls from './Controls';

describe('Controls', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Controls />);
  });

  it('Should match Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});