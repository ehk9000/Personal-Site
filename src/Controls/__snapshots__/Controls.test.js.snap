// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controls Should match Snapshot 1`] = `undefined`;
